starters/vue3-xstate-css/src/views/HomeView.vue

```diff
@@ -1,9 +1,35 @@
-<script setup lang="ts">
-import TheWelcome from '../components/TheWelcome.vue';
-</script>
+<script setup lang="ts"></script>
 
 <template>
   <main>
-    <TheWelcome />
+    <div class="title">
+      <h1>Vue 3 + XState + CSS</h1>
+    </div>
+    <div class="links">
+      <RouterLink to="/counter">See button example component</RouterLink>
+      <RouterLink to="/fetch">See data fetching example component</RouterLink>
+    </div>
   </main>
 </template>
+
+<style scoped>
+.title {
```

Comment:
Also looks like a potentially shared style, could refactor the style into a separate file perhaps?

Reply:
This one is also meant only for the title on this home page. Perchance the class names are a little too generic? I could adjust that so it's clearer they're specifically for this page. 😁